scripts/poc906_deploy_cryptopunks_nft.py

```diff
@@ -0,0 +1,491 @@
+        {"from": owner}
+    )
+
+    loans_peripheral_weth.setLiquidationsPeripheralAddress(
+        liquidations_peripheral,
+        {"from": owner}
+    )
+    loans_peripheral_weth.setLiquidityControlsAddress(
+        liquidity_controls,
+        {"from": owner}
+    )
+
+    liquidations_core.setLiquidationsPeripheralAddress(
+        liquidations_peripheral,
+        {"from": owner}
+    )
+
+    liquidations_peripheral.addLoansCoreAddress(
+        weth,
+        loans_core_weth,
+        {"from": owner}
+    )
+    liquidations_peripheral.addLendingPoolPeripheralAddress(
+        weth,
+        lending_pool_peripheral_weth,
+        {"from": owner}
+    )
+    liquidations_peripheral.setCollateralVaultPeripheralAddress(
+        collateral_vault_peripheral,
+        {"from": owner}
+    )
+
+    loans_peripheral_weth.addCollateralToWhitelist(cool_cats_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(hashmasks_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(kennel_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(doodles_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(wow_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(mutant_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(veefriends_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(pudgypenguins_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(bayc_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(wpunks_instance, {"from": owner})
+    loans_peripheral_weth.addCollateralToWhitelist(cryptopunks_instance, {"from": owner})
+
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, cryptopunks_instance, web3.toWei(750, "ether"))  # TBC
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, wpunks_instance, web3.toWei(750, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, bayc_instance, web3.toWei(500, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, doodles_instance, web3.toWei(50, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, pudgypenguins_instance, web3.toWei(35, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, mutant_instance, web3.toWei(75, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, veefriends_instance, web3.toWei(35, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, kennel_instance, web3.toWei(35, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, wow_instance, web3.toWei(25, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, cool_cats_instance, web3.toWei(25, "ether"))
+    liquidity_controls.changeMaxCollectionBorrowableAmount(True, hashmasks_instance, web3.toWei(5, "ether"))
```

Comment:
This doesn't need to be here, right? Or is it here just because we want to perform this deployment together with the one coming from #18 ?

Reply:
It does not. Removed.